### ISSUE TITLE
UI editor does not take the full window size

### ISSUE THREAD

**Issue by gr8web** [onivim/oni2#2776]
Hello,

I installed onivim on my macos 10.13.6 (17G14042). When started it takes only part of the current window.

![xef9jsafd1261](https://user-images.githubusercontent.com/1122741/100527061-34472080-31cf-11eb-8762-8d2ea24ebea1.gif)

As can be seen on the gif the editor does not scale to the full window.
I tried the 0.5 stable build and a nightly one.

I did not have the issue with 0.3 version which I had before.

**Comment by mirsella** [comment]
to have log you can launch onivim like this : `Oni2 --nofork --debug` you can then put the output on a pastebin maybe and link it there.

**Comment by gr8web** [comment]
https://pastebin.com/ZdmNktdJ

Just realized that the issue appears with plugged external displays.
When I unpug external displays from my macbook the issue disappears.

**Comment by CrossR** [comment]
This sounds like an issue where the UI is being shown on one monitor, but the size/resolution of the other monitor is being used for rendering.

Was Oni2 being launched on the secondary display (i.e. not whichever one you've set as the primary/main in macOS)?

**Comment by gr8web** [comment]
No, it was launched on the main one (the primary/main in macOS). When I move the Oni window to my laptop screen it looks fine and when I move it over to the plugged in external displays it breaks.

**Comment by bryphe** [comment]
This looks really similar to #2564 - which I've been trying to hunt down a repro for - it is interesting that it is only reproducing with the plugged in external display. Will try reproducing again with that info.

**Comment by gr8web** [comment]
> This looks really similar to #2564 - which I've been trying to hunt down a repro for - it is interesting that it is only reproducing with the plugged in external display. Will try reproducing again with that info.

Indeed it does. I dont know whether it is somehow directly related to the external displays because the 0.3 was looking fine. Thank you for having a look.

**Comment by bryphe** [comment]
I'm able to reproduce this as now with OSX 10.13 and an external display - thanks @gr8web ! I'm continuing to investigate

**Comment by Xemanth** [comment]
Is this fixed?
